specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/examples/changeTenantPolicy.json

```diff
@@ -0,0 +1,30 @@
+{
+  "parameters": {
+    "api-version": "2019-10-01-preview",
+    "body": {
+      "blockSubscriptionsLeavingTenant": true,
+      "blockSubscriptionsIntoTenant": true,
+      "exemptedPrincipals": [
+        "e879cf0f-2b4d-5431-109a-f72fc9868693",
+        "9792da87-c97b-410d-a97d-27021ba09ce6"
+      ]
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "291bba3f-e0a5-47bc-a099-3bdcb2a50a05",
+        "name": "Default",
+        "type": "Microsoft.Subscription/Policies/default.",
```

Comment:
>. [](start = 56, length = 1)

The type should be `Microsoft.Subscription/policies`

Reply:
The comments in this file apply to all the examples.

---
In reply to: [553070641](https://github.com/Azure/azure-rest-api-specs/pull/11672#discussion_r553070641) [](ancestors = 553070641)